Added additional info about deleting stored payment method (#1473)
magento/merchdocs: src/customers/account-dashboard-stored-payment-methods.md

1. Clicks <span class="btn">Place Order</span>.

The saved payment method is then displayed in the _Stored Payment Methods_ tab of the customer dashboard.

## Delete a stored payment method

> Any of added earlier stored payment methods can not be edited by customer, but can be deleted only. This action cannot be undone.

Reviewer: ```suggestion
Any previously added, stored payment methods cannot be edited by the customer, they can only be deleted. This action cannot be undone.
```

Author: Thank you for the review. Changes applied
